libmscore/layout.cpp

```diff
@@ -1856,108 +1857,117 @@ System* Score::getNextSystem(LayoutContext& lc)
 
 //---------------------------------------------------------
 //   createMMRest
-//    create a multi measure rest from m to lm (inclusive)
+//    create a multimeasure rest
+//    from firstMeasure to lastMeasure (inclusive)
 //---------------------------------------------------------
 
-void Score::createMMRest(Measure* m, Measure* lm, const Fraction& len)
+void Score::createMMRest(Measure* firstMeasure, Measure* lastMeasure, const Fraction& len)
 {
-    int n = 1;
-    if (m != lm) {
-        for (Measure* mm = m->nextMeasure(); mm; mm = mm->nextMeasure()) {
-            ++n;
-            mm->setMMRestCount(-1);
-            if (mm->mmRest()) {
-                undo(new ChangeMMRest(mm, 0));
-            }
-            if (mm == lm) {
+    int numMeasuresInMMRest = 1;
+    if (firstMeasure != lastMeasure) {
+        for (Measure* m = firstMeasure->nextMeasure(); m; m = m->nextMeasure()) {
+            ++numMeasuresInMMRest;
+            m->setMMRestCount(-1);
+            if (m->mmRest()) {
+                undo(new ChangeMMRest(m, 0));
+            }
+            if (m == lastMeasure) {
```

Comment:
This is type of thing I mean when I say I could use a comment to save the trouble of checking whether you actually changed anything here or just renamed variables.

Reply:
Ah, yes. Tbh, I should rewind the commit history and make that a separate commit again.

Reply:
History has been rewritten.